scripts/globals/magic_utils/spell_damage.lua

```diff
@@ -0,0 +1,883 @@
+            magicAcc = magicAcc + 256 -- Need citation. 256 seems OP
+        end
+
+        -- Add acc for skillchains
+        if skillchainCount > 0 then
+            magicAcc = magicAcc + 25
+        end
+
+        -----------------------------------
+        -- magicAcc from Job Points.
+        -----------------------------------
+        -- WHM Job Points
+        if casterJob == xi.job.WHM then
+            magicAcc = magicAcc + caster:getJobPointLevel(xi.jp.WHM_MAGIC_ACC_BONUS)
+
+        -- BLM Job Points
+        elseif casterJob == xi.job.BLM then
+            magicAcc = magicAcc + caster:getJobPointLevel(xi.jp.BLM_MAGIC_ACC_BONUS)
+
+        -- RDM Job Points
+        elseif casterJob == xi.job.RDM then
+            -- RDM Job Point: During saboteur, Enfeebling MACC +2
+            if skillType == xi.skill.ENFEEBLING_MAGIC and caster:hasStatusEffect(xi.effect.SABOTEUR) then
+                magicAcc = magicAcc + (caster:getJobPointLevel(xi.jp.SABOTEUR_EFFECT)) * 2
+            end
+            -- RDM Job Point: Magic Accuracy Bonus, All MACC + 1
+            magicAcc = magicAcc + caster:getJobPointLevel(xi.jp.RDM_MAGIC_ACC_BONUS)
+
+        -- NIN Job Points
+        elseif casterJob == xi.job.NIN then
+            -- NIN Job Point: Ninjitsu Accuracy Bonus
+            if skillType == xi.skill.NINJUTSU then
+                magicAcc = magicAcc + caster:getJobPointLevel(xi.jp.NINJITSU_ACC_BONUS)
+            end        
+
+        -- SCH Job Points
+        elseif casterJob == xi.job.SCH then
+            if (spellGroup == xi.magic.spellGroup.WHITE and caster:hasStatusEffect(xi.effect.PARSIMONY)) or
+                (spellGroup == xi.magic.spellGroup.BLACK and caster:hasStatusEffect(xi.effect.PENURY))
+            then
+                magicAcc = magicAcc + caster:getJobPointLevel(xi.jp.STRATEGEM_EFFECT_I)
+            end
+        end
+
+        -----------------------------------
+        -- magicAcc from Merits.
+        -----------------------------------
+        -- BLM Merits
+        if casterJob == xi.job.BLM and skillType == xi.skill.ELEMENTAL_MAGIC then
+            magicAcc = magicAcc + caster:getMerit(xi.merit.ELEMENTAL_MAGIC_ACCURACY)
+
+        -- RDM Merits
+        elseif casterJob == xi.job.RDM and spellElement >= xi.magic.element.FIRE and spellElement <= xi.magic.element.WATER then
+            magicAcc = magicAcc + caster:getMerit(rdmMerit[spellElement])
+
+        -- NIN Merits
+        elseif casterJob == xi.job.NIN and skillType == xi.skill.NINJUTSU then
+            magicAcc = magicAcc + caster:getMerit(xi.merit.NIN_MAGIC_ACCURACY)
+
+        -- BLU Merits
+        elseif casterJob == xi.job.BLU and skillType == xi.skill.BLUE_MAGIC then
+            magicAcc = magicAcc + caster:getMerit(xi.merit.MAGICAL_ACCURACY)
+        end
+        -- OTHER merits. ???? which are...? Modifiers need a rework/reorganization
+        magicAcc = magicAcc + caster:getMerit(xi.merit.MAGIC_ACCURACY)
```

Comment:
Where is this generic `xi.merit.MAGIC_ACCURACY` coming from?

Reply:
As I pointed out in the comment, I have no clue XD
It was there in the original magic.lua file and decided to keep it in case someone had more info on it.
It is on my "to destroy if no info is provided" private list

Reply:
its a modifier set by reading the [merit table](https://github.com/LandSandBoat/server/blob/base/sql/merits.sql#L226). Whether or not you can [actually merit it ](https://github.com/LandSandBoat/server/blob/base/sql/char_merit.sql)is another story.

The code fr applying merits is rather generic so you won't find core code doing application of the specific merit.

Reply:
oh, I see. It's RDM category 2 merit